### ISSUE TITLE
[0.1.7] `default-scenarios` plugin fails to activate

### ISSUE THREAD

**Issue by suganya-sk** [jupyterlab/ui-profiler#38]
## Description

The latest version 0.1.7 has the following errors in console - 
![image](https://user-images.githubusercontent.com/19615498/227887933-42a78fc4-dba0-4a2f-b72e-0778bc56d648.png)

## Reproduce

1. Install ui-profiler 0.1.7
2. Start Jupyterlab (3.6.2) from the same venv
3. The errors mentioned above show up in the browser console.

## Expected behavior

No errors in activating the plugins included in the latest version.

## Note
While debugging this, I was unable to install a local copy of this. The instructions in README include `pip install -e`. But I ran into an error - 

> distutils.errors.DistutilsOptionError: No configuration found for dynamic 'description'.
>     Some dynamic fields need to be specified via `tool.setuptools.dynamic`
>     others must be specified via the equivalent attribute in `setup.py`.

and tracked this down to this issue with pip + pyproject.toml  - https://github.com/pypa/pip/issues/11393. To confirm, I removed `pyproject.toml` and re-attemped the src installation, a somewhat-incomplete version was installed successfully.

@krassowski Could you please advise how you achieve a source installation of this package? 

## Context

- JupyterLab version: 3.6.2

**Comment by krassowski** [comment]
> While debugging this, I was unable to install a local copy of this.

What version of pip did you use when you encountered the installation error (`pip --version`)? I was able to reproduce the installation error when using `pip install -e .` with an older version of pip (20.2.3) but I was not able to reproduce it with newer versions (I tested 22.3.1 and 23.0.1 in two separate environments). I think this is a pip error which was resolved since (and it all works smoothly on CI).

> Could you please advise how you achieve a source installation of this package?

Just to clarify, are you using the following command?

```
jupyter labextension install @jupyterlab/ui-profiler
```

Strangely, I was not able to reproduce it but based on the error message I believe I see where the issue is originating from. I am just perplexed as to why it works for me.
